bases/br_inep_sinopse_estatistica_educacao_basica/etapa_ensino_serie/table_config.yaml

```diff
@@ -0,0 +1,245 @@
+# Igual ao dataset.name mas como lower case.
+# Exemplos: br_ibge_populacao, br_inep_censo_escolar
+dataset_id: br_inep_sinopse_estatistica_educacao_basica
+
+table_id: etapa_ensino_serie
+
+# Título da tabela.
+title: Matrículas por etapa ensino e série
+
+# Descreva a tabela. Essas são as primeiras frases que um usuário vai ver.
+# Você não precisa ser muito conciso. Sinta-se a vontade para dar exemplos de
+# como usar os dados.
+# Se souber, liste também aplicações: pesquisa, apps, etc. que usem os dados.,
+description: A base conta com o total de matrículas por município para todas as etapas e subetapas de ensino, além da discriminação dos anos para Anos Iniciais e Finais do Ensino Fundamental, bem como as série do Ensino Médio
+
+# As máximas unidades espaciais que a tabela cobre.
+# Exemplo:
+#    - sa.br
+#  
+#    - sa.br.sp
+#  
+#    - world
+spatial_coverage:
+    - sa.br
+
+# Anos cobertos pela tabela.
+# Exemplos:
+#     - 1995(1)2019
+# Caso a cobertura não seja contínua:
+#     - 2002(2)2010
+#     - 2016
+#     - 2020
+temporal_coverage:
+    - 2007(1)2022
+
+# A unidade temporal com qual a tabela é atualizada.
+# Opções em 'https://basedosdados.org/api/3/action/bd_available_options'
+update_frequency: one_year
+
+# Nível de observação da tabela: o que representa cada linha.
+# A combinação das colunas aqui deve construir uma chave única da tabelaOpções de entity em 'https://basedosdados.org/api/3/action/bd_available_options'
+# Caso a entidade seja espacial incluir a informação de 'country' Exemplos:
+#     - entity: year
+#       columns:
+#           - ano
+#     - country: br
+#       entity: state
+#       columns:
+#           - sigla_uf
+observation_level:
+    - entity: year
+      columns:
+        - ano
+    - country: br
+      entity: municipality
+      columns:
+        - id_municipio
+    - entity: other
+      columns:
+        - rede
+    - entity: other
+      columns:
+        - etapa_ensino
+    - entity: other
+      columns:
+        - serie
+
+last_updated:
+    metadata: '2023-04-05'
+    data: '2023-04-05 10:13:00'
+    release: '2023-04-05 10:13:00'
+
+# Versão da tabela. Seguindo o padrão de semantic versioning.
+# Exemplos: v1.0, v1.1.3
+version: v2.0
+
+# Quem está preenchendo esses metadados?
+published_by:
+    name: Lucas Moreira
+    email: lucas.moreira@basedosdados.org
+    github_user: lucasnascm
+    ckan_user: lucasnascm
+    website:
+
+# Qual organização/departamento/pessoa tratou os dados?
+# As vezes há um ponto intermediário entre os dados originais e subir na Base dos Dados.
+# Se essa pessoa é você, preencha abaixo com suas informações.
+data_cleaned_by:
+    name: Lucas Moreira
+    email: lucas.moreira@basedosdados.org
+    github_user: lucasnascm
+    ckan_user: lucasnascm
+    website:
+
+# Se houve passos de tratamento, limpeza e manipulação de dados, descreva-os aqui.
+data_cleaning_description: Junção das diferentes planilhas originais, reshape pro formato long, partição por UF
```

Comment:
Estamos tentando padronizar essa parte de descrição, aí seria bom numerar as etapas. Acho bacana pra facilitar pro usuário e pra gente mesmo ter mais clareza do que aconteceu. Vou colocar aqui o exemplo da estban que to usando como referencia pra todos:

data_cleaning_description: 1. Retirar acentos da coluna de instituicoes; 2. Criar coluna id_municipio; 3. Dropar colunas 'MUNICIPIO', 'CODMUN_IBGE', 'CODMUN' originais da base descenessarias ou substituidas; 4. Tranformar a base do formato wide para o formato long para transformar 55 colunas de verbetes em uma única; 5. Corrigir unidades monetárias; 6. Criar regex para extrair identificador dos verbetes; 7. Criar colunas de ano e mês